### PR TITLE
Add latlng field, generate latlng and regionId from address

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

File: src/db/seeds/UpdateHawkerCentre.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    return '2';
+  } else if (isInWest) {
+    return '3';
+  } else if (isInCentral) {
+    return '4';
+  } else if (isInNorthEast) {
+    return '5';
+  } else {
+    //Will never enter here
+    return '1';
```
Comment (wenhaogoh) [inline src/db/seeds/UpdateHawkerCentre.ts]: Perhaps should throw an error here?

File: src/consts/index.ts [hunk 2]
```diff
@@ -26,4 +26,6 @@ export const GCS_CLIENT_EMAIL = process.env.GCS_CLIENT_EMAIL || '';
 export const GCS_PRIVATE_KEY = process.env.GCS_PRIVATE_KEY || '';
 export const GCS_BASE_URL = 'https://storage.googleapis.com';
 
+export const GOOGLE_MAPS_API_KEY = process.env.GOOGLE_MAPS_API_KEY || '';
```
Comment (ivantjh) [inline src/consts/index.ts:29]: Remember to add to Heroku the env variable

File: src/db/seeds/UpdateHawkerCentre.ts [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+2 | East | 46-52 && 81
+3 | West | 60-71
+4 | Central | 01-45 && 58-59
+5 | NorthEast | 53-57 && 79-80
+*/
+
+const getRegionId = (address: string) => {
+  const postalCode = address.substring(address.indexOf('S(') + 2);
+  const firstTwoDigits = parseInt(postalCode.substring(0, 2));
+  const isInNorth: Boolean = firstTwoDigits <= 78 && firstTwoDigits >= 72 && firstTwoDigits !== 74;
```
Comment (ivantjh) [inline src/db/seeds/UpdateHawkerCentre.ts]: No need type parameter, typescript can infer. Anyway, typescript types are usually written in lowercase. More info [here](https://www.typescriptlang.org/docs/handbook/basic-types.html#about-number-string-boolean-symbol-and-object)

File: src/consts/index.ts (hunk 2 above)
Comment (oliviajohansen) [reply src/consts/index.ts:29]: ok done, also added to github secrets

File: src/db/seeds/UpdateHawkerCentre.ts (hunk 3 above)
Comment (oliviajohansen) [reply src/db/seeds/UpdateHawkerCentre.ts]: noted, removed thanks

File: src/db/seeds/UpdateHawkerCentre.ts (hunk 1 above)
Comment (oliviajohansen) [reply src/db/seeds/UpdateHawkerCentre.ts]: ok done

**Pull request merged**